pages/mesosphere/dcos/1.14/installing/production/deploying-dcos/installation/index.md

```diff
@@ -353,6 +362,11 @@ At this point your directory structure should resemble:
     ```
 
 3.  <A name="masterinstall"></A> Run the following commands on each of your master nodes in succession to install DC/OS using your custom build file:
+    * If created, copy the pre-shared key to your master nodes at /var/lib/dcos/dcos-bootstrap-ca-psk
+
+        ```bash
+        scp genconf/ca/psk <master-ip>:/var/lib/dcos/dcos-bootstrap-ca-psk
```

Comment:
Above we have `genconf/psk`, here we say `genconf/ca/psk`

Reply:
Good catch